system_tests/bigquery.py

```diff
@@ -90,7 +92,15 @@ def test_update_dataset(self):
         after = [grant for grant in dataset.access_grants
                  if grant.entity_id != 'projectWriters']
         dataset.access_grants = after
-        dataset.update()
+
+        # We need to wait for the changes in the dataset to propgate and be
+        # eventually consistent. The alternative outcome is a 403 Forbidden
+        # response from upstream due to upstream rate limiting.
```

Comment:
1. Still says "propgate"
1. Still implies that eventual consistency has anything to with this failure, which doesn't seem to be true.